misk-aws/src/main/kotlin/misk/dynamodb/RealDynamoDbModule.kt

```diff
@@ -0,0 +1,46 @@
+package misk.dynamodb
+
+import com.amazonaws.auth.AWSCredentialsProvider
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapperConfig
+import com.google.inject.Provides
+import misk.cloud.aws.AwsRegion
+import misk.inject.KAbstractModule
+import javax.inject.Singleton
+
+internal class RealDynamoDbModule : KAbstractModule() {
```

Comment:
why is this internal?

Reply:
Oops. It was internal when I was developing for my needs. Will fix